staging/src/k8s.io/apimachinery/pkg/util/clock/clock_test.go

```diff
@@ -25,12 +25,6 @@ var (
 	_ = Clock(RealClock{})
 	_ = Clock(&FakeClock{})
 	_ = Clock(&IntervalClock{})
-
-	_ = Timer(&realTimer{})
-	_ = Timer(&fakeTimer{})
-
-	_ = Ticker(&realTicker{})
-	_ = Ticker(&fakeTicker{})
 )
```

Comment:
We probably no longer need this whole test file anymore, given we only reference other implementation.